iplant/webapps.gradle

```diff
@@ -112,15 +112,15 @@ project(':de-webapp') {
     }
 
     task draftBootRepackage(type: BootRepackage, dependsOn: draftWar) {
-        dependsOn webpack
+        dependsOn webpackDev
```

Comment:
The changes to this file shouldn't be necessary (see comments below for `package.json`).